examples/adc_dma.rs

```diff
@@ -1,4 +1,4 @@
-//! Example of reading ADC1 data using DMA1/2
+//! Example of reading ADC1 data using DMA1/2 through DMA
```

Comment:
I think this was fine before

```suggestion
//! Example of reading ADC1 data using DMA1
```

Reply:
Thanks, corrected now